src/snovault/helpers/helper.py

```diff
@@ -268,18 +268,16 @@ def build_terms_filter(query_filters, field, terms):
         query_filters['must'].append(filter_condition)
 
 
-def set_filters(request, query, result, static_items=None, filter_exclusion=None):
-    """
-    Sets filters in the query
-    """
+def set_filters(request, query, static_items=None, filter_exclusion=None):
+    """Set filters in the query."""
+    # pylint: disable=too-many-local-variables
     query_filters = query['post_filter']['bool']
     used_filters = {}
-    if static_items is None:
-        static_items = []
+    result_filters = []
 
     # Get query string items plus any static items, then extract all the fields
     qs_items = list(request.params.items())
-    total_items = qs_items + static_items
+    total_items = qs_items + (static_items or [])
```

Comment:
I'm not a fan of the ```+``` for list create/concatenation unless it is being used a lot in the function.  Here it's just a shortcut to create a new list from two other lists. 

I'd prefer to make it more readable with explicit steps.
```
total_items = qs_items.copy()
if static_items:
    total_items.extend(static_items)
```
    
The original code was fine too, so may be best to not change in any case.

Reply:
Agreed. Your way seems more pythonic. However, the code works and we are not trying to polish up set_filters. So it is best to leave it.

Reply:
Hey, We need to return this to it's original form.  It was an unnecessary change and we're being more strict on that now.